fetch_news.py

```diff
@@ -156,6 +153,20 @@ def get_mean_embedding(mongo_client = None, connection_string = None, classifica
 
     return m
 
+def delete_old_article_text(mongo_client):
+    candidates_collection = mongo_client['aiidprod'].candidates
+    for article in candidates_collection.find({'text': {'$exists': True}}):
+      article_date = dateutil.parser.parse(
```

Comment:
please double-check the indenting/tab spacing? Looks like it should be 4 spaces in this file everywhere. This line and others below look like 2-space.